docs/docusaurus.config.js

```diff
@@ -119,6 +119,63 @@ const config = {
                             },
                         ],
                     },
+                    {
+                        title: 'How-to guides',
+                        items: [
+                            {
+                                label: 'Angular Feature Flags',
```

Comment:
Is it necessary to have "Feature Flags" on each label? It looks visually noisy IMO but if there's truly a SEO benefit I'm guessing it's okay.

Reply:
I think there probably is an SEO benefit here, but I will check on this as it's a good point.